src/lib/php/libschema.php

```diff
@@ -383,7 +383,8 @@ function dropIndexes()
       }
       foreach ($IndexInfo as $name => $sql)
       {
-        if (empty($name) || $this->schema['INDEX'][$table][$name] == $sql)
+        if (empty($name) || $this->schema['INDEX'][$table][$name] == $sql ||
+             (str_replace($table." ", "public.".$table." ", $this->schema['INDEX'][$table][$name]) == $sql))
```

Comment:
Won't it be good to use str_replace at line no. 889 inside `addIndexes()` so that the call will be reduced to only once?

Reply:
No. Only PSQL 9.5 or above  needs replace with "Public".  i think most of the people are still using older versions. so both the checks are required.

Reply:
What I am saying is that instead of replacing the string during the check, if we replace it at the time of assignment, we can save one `str_replace` call.